### DISCUSSION TITLE
Multiple graphql endpoints

### DISCUSSION THREAD

**Discussion by adamjw3** [vercel/next.js#16428] [Help]
Looking for a best practice example on implementing multiple graphql endpoints. Both server and client side examples.

https://headlesscmsgraphqlendpoint
https://someotherservicegraphqlendpoint

Looked at apollo client and and it only excepts one end point 

```
import ApolloClient from "apollo-boost";

const client = new ApolloClient({
 uri: "https://headlesscmsgraphql"
});
```

Thinking ideally want to handle this in the new api folder of nextjs, is that best practice.

Thanks

**Comment by ManAnRuck** [comment]
@adamjw3 did you find a solution?
